server/admin-function/check-admin.ts

```diff
@@ -0,0 +1,21 @@
+"use server";
+
+import { dbAdmin } from "@/lib/firebase-admin";
+
+export const checkAdmin = async (domain: string, loggedInUserId:string): Promise<boolean> => {
+  try {
+
+    console.log('checkAdmin:', domain, loggedInUserId);
```

Comment:
remove console

Reply:
even on server side?

Reply:
yes it's always good pratice to cleanup console.log or else it becomes unreadable